client/src/App.tsx

```diff
@@ -13,6 +13,22 @@ import { SnackBarProvider } from './context/useSnackbarContext';
 import { MuiPickersUtilsProvider } from '@material-ui/pickers';
 import DateFnsUtils from '@date-io/date-fns';
 
+import { TourProvider } from '@reactour/tour';
+const steps = [
+  {
+    selector: '[data-tour="step-1"]',
+    content: 'you can see your booking from here',
+  },
+  {
+    selector: '[data-tour="step-2"]',
+    content: 'you can click here to edit your profile',
+  },
+  {
+    selector: '[data-tour="step-3"]',
+    content: 'you can click here to add a picture of your self',
+  },
+];
```

Comment:
We should move this to a more defined location for better maintainability, I don't want to bloat the App.tsx with this when we need to expand product tours.